From: Qingqing <wangyiqing@kylinos.cn>
Date: Wed, 6 Aug 2025 07:29:29 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 100.0% (33 of 33 strings)

Translation: openkylin-nile-sp2/kylin-file-crush
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-sp2/kylin-file-crush/vi/
---
 .../file-smash/translations/kylin-file-crush_vi.ts | 54 +++++++++++-----------
 1 file changed, 27 insertions(+), 27 deletions(-)

diff --git a/treasure-chest/file-smash/translations/kylin-file-crush_vi.ts b/treasure-chest/file-smash/translations/kylin-file-crush_vi.ts
index b88f50c..3a36a0f 100644
--- a/treasure-chest/file-smash/translations/kylin-file-crush_vi.ts
+++ b/treasure-chest/file-smash/translations/kylin-file-crush_vi.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../alertdialog.ui" line="14"/>
         <source>Form</source>
-        <translation>Định dạng</translation>
+        <translation>Mẫu</translation>
     </message>
 </context>
 <context>
@@ -14,7 +14,7 @@
     <message>
         <location filename="../shreddialog.cpp" line="977"/>
         <source>Click the “+” button or directly drag and drop files here</source>
-        <translation>Bấm nút &quot;+&quot; hoặc kéo thả tập tin vào đây</translation>
+        <translation>Nhấp vào nút &quot;+&quot; hoặc kéo và thả tệp vào đây</translation>
     </message>
 </context>
 <context>
@@ -22,12 +22,12 @@
     <message>
         <location filename="../kalertdialog.cpp" line="47"/>
         <source>The file you selected is a directory or does not have permissions, cannot be shredded!</source>
-        <translation>Tập tin bạn chọn là thư mục hoặc không có quyền, không thể hủy!</translation>
+        <translation>Tệp bạn đã chọn là một thư mục hoặc không có quyền hạn, không thể hủy được !</translation>
     </message>
     <message>
         <location filename="../kalertdialog.cpp" line="50"/>
         <source>sure</source>
-        <translation>Đồng ý</translation>
+        <translation>Xác nhận</translation>
     </message>
 </context>
 <context>
@@ -35,17 +35,17 @@
     <message>
         <location filename="../ksuredialog.cpp" line="48"/>
         <source>Are you sure to start crushing?</source>
-        <translation>Bạn có chắc chắn muốn bắt đầu hủy?</translation>
+        <translation>Bạn có chắc chắn muốn bắt đầu hủy không?</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="54"/>
         <source>Crushed files cannot be recovered!</source>
-        <translation>Tập tin sau khi hủy sẽ không thể khôi phục!</translation>
+        <translation>Các tệp bị hủy sẽ không thể khôi phục được!</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="62"/>
         <source>sure</source>
-        <translation>Đồng ý</translation>
+        <translation>Xác nhận</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="70"/>
@@ -73,7 +73,7 @@
     <message>
         <location filename="../shreddialog.cpp" line="113"/>
         <source>No file selected to be shredded</source>
-        <translation>Chưa chọn tập tin cần hủy</translation>
+        <translation>Chưa chọn tệp muốn hủy</translation>
     </message>
     <message>
         <source>The file is completely shredded and cannot be recovered</source>
@@ -83,13 +83,13 @@
         <location filename="../shreddialog.cpp" line="38"/>
         <location filename="../shreddialog.cpp" line="39"/>
         <source>File Crush</source>
-        <translation>Công cụ hủy tập tin</translation>
+        <translation>Máy hủy tệp</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="65"/>
         <location filename="../shreddialog.cpp" line="815"/>
         <source>File shredding, please wait a moment</source>
-        <translation>Đang hủy tập tin, vui lòng đợi</translation>
+        <translation>Đang hủy tệp, vui lòng chờ đợi</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="131"/>
@@ -104,42 +104,42 @@
     <message>
         <location filename="../shreddialog.cpp" line="227"/>
         <source>File Path</source>
-        <translation>Đường dẫn</translation>
+        <translation>Đường dẫn tệp</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="394"/>
         <source>Loading file information... </source>
-        <translation>Đang tải thông tin tập tin... </translation>
+        <translation>Đang tải thông tin tệp... </translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="577"/>
         <source>Completely crush files or folders that cannot be deleted</source>
-        <translation>Hủy hoàn toàn các tập tin/thư mục không thể xóa</translation>
+        <translation>Hủy hoàn toàn các tệp hoặc thư mục không thể xóa được</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="578"/>
         <source>Shred File</source>
-        <translation>Hủy tập tin</translation>
+        <translation>Hủy</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="579"/>
         <source>Deselect</source>
-        <translation>Bỏ chọn</translation>
+        <translation>Xóa bỏ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="580"/>
         <source>Add File</source>
-        <translation>Thêm tập tin</translation>
+        <translation>Thêm tệp</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="581"/>
         <source>Note:File shredding cannot be cancelled, please exercise caution!</source>
-        <translation>Lưu ý: Không thể hủy bỏ thao tác hủy tập tin!</translation>
+        <translation>*Quá trình hủy không thể hủy , vui lòng thao tác cẩn thận!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="781"/>
         <source>File shredding completed</source>
-        <translation>Đã hủy tập tin thành công</translation>
+        <translation>Hoàn tất việc hủy tệp</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="834"/>
@@ -154,24 +154,24 @@
     <message>
         <location filename="../shreddialog.cpp" line="835"/>
         <source> File shredding failed</source>
-        <translation> Hủy tập tin thất bại</translation>
+        <translation> Việc hủy tệp thất bại!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="837"/>
         <location filename="../shreddialog.cpp" line="840"/>
         <source>cancel</source>
-        <translation>Hủy bỏ</translation>
+        <translation>Hủy</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="838"/>
         <location filename="../shreddialog.cpp" line="843"/>
         <source>view info</source>
-        <translation>Xem chi tiết</translation>
+        <translation>Thông tin chi tiết</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="886"/>
         <source>Please select the file to cacel!</source>
-        <translation>Vui lòng chọn tập tin cần bỏ!</translation>
+        <translation>Vui lòng chọn tập tin để xóa bỏ!</translation>
     </message>
     <message>
         <source>Select file</source>
@@ -196,7 +196,7 @@
     <message>
         <location filename="../shreddialog.cpp" line="858"/>
         <source>Please select the file to shred!</source>
-        <translation>Vui lòng chọn tập tin cần hủy!</translation>
+        <translation>Vui lòng chọn tệp để hủy!</translation>
     </message>
     <message>
         <source>You did not select a file with permissions!</source>
@@ -208,7 +208,7 @@
     <message>
         <location filename="../shredfailinfodialog.cpp" line="22"/>
         <source>cancel</source>
-        <translation>Hủy bỏ</translation>
+        <translation>Hủy</translation>
     </message>
     <message>
         <location filename="../shredfailinfodialog.cpp" line="100"/>
@@ -218,7 +218,7 @@
     <message>
         <location filename="../shredfailinfodialog.cpp" line="100"/>
         <source>File Path</source>
-        <translation>Đường dẫn</translation>
+        <translation>Đường dẫn tệp</translation>
     </message>
 </context>
 <context>
@@ -226,12 +226,12 @@
     <message>
         <location filename="../shredmanager.cpp" line="49"/>
         <source>Shred Manager</source>
-        <translation>Quản lý hủy tập tin</translation>
+        <translation>Máy hủy tệp</translation>
     </message>
     <message>
         <location filename="../shredmanager.cpp" line="54"/>
         <source>Delete files makes it unable to recover</source>
-        <translation>Tập tin sau khi hủy sẽ không thể phục hồi</translation>
+        <translation>Các tệp bị hủy sẽ không thể khôi phục được!</translation>
     </message>
 </context>
 </TS>
